src/main/java/com/mkopec/ticketbookingapp/mapper/ReservationMapper.java

```diff
@@ -0,0 +1,24 @@
+package com.mkopec.ticketbookingapp.mapper;
+
+import com.mkopec.ticketbookingapp.domain.Reservation;
+import com.mkopec.ticketbookingapp.dtos.ReservationPostDTO;
+import com.mkopec.ticketbookingapp.dtos.ShortReservationDTO;
+import org.mapstruct.Mapper;
+import org.mapstruct.Mapping;
+import org.mapstruct.Mappings;
+import org.springframework.beans.factory.annotation.Autowired;
+
+@Mapper(componentModel = "spring")
+public abstract class ReservationMapper {
```

Comment:
jakie są plusy/minusy korzystania z takich mapperów?

Reply:
Plusy:
- sami nie musimy ich implementować (są generowane)
- nawet jak zrobimy jakąś literówkę w nazwach to już przy kompilacji pojawi się błąd 
Minusy:
- problem przy mapowaniu obiektów złożonych (nie zawsze domyślne mapowanie nam pasuje, wtedy sami musimy wstrzykiwać mapper, no i żeby wywołać mapowanie musimy wtedy źródło wyrazić jako expression)
- po mapowaniu może pojawić się w niektórych miejscach null (np kiedy nazwy pomiędzy obiektami się różnią, wtedy sami musimy sprawdzić czy nasz mapper dobrze działa)

Reply:
* jak zrobimy literówkę w nazwach i robimy mapowanie sami to nie będzie błędu kompilacji?
* czy samo testowanie mappera ma sens?

Reply:
Jeśli zrobimy literówkę np taką że odwołujemy się do source którego nie ma, albo metody której nie ma błąd będzie już podczas kompilacji, gorzej kiedy z literówką trafimy na source. Jeśli zrobimy mapowanie sami, np za pomocą expression też błąd powinien wyskoczyć podczas kompilacji.
Tak, testowanie mappera ma sens jak najbardziej